### DISCUSSION TITLE
Confused | Where is the timeline for the externship and project details ?

### DISCUSSION THREAD

**Discussion by tushar5526** [github/india#589] [Students]
Where is the timelines regarding the deadlines of project submission and project ideas ?

**Comment by Isha1-n** [comment]
Yes after the procedure of sending otp in mail. Profile goes under review and now it is is still not processed.  Kindly guide

**Reply by tejaswaG2203** [reply]
it takes 48-72 hours to get reviewed.

**Reply by punitjain03** [reply]
Externship starts from Jan 10 to 3 months

**Reply by vivmost** [reply]
oops😅, actually I was posting the ques. as a new thread but thanks!

**Reply by tushar5526** [reply]
Any information on the project submission guidelines and deadlines to select project would be really helpful.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
